x/project/spec/03_messages.md

```diff
@@ -0,0 +1,149 @@
+# Messages
+
+In this section we describe the processing of the bonds messages and the corresponding updates to the state. All created/modified state objects specified by each message are defined within the [state](./02_state.md) section.
+
+## MsgCreateProject
+
+Project can be created by any address using `MsgCreateProject`.
+
+| **Field**              | **Type**           | **Description**                                                                                               |
+|:-----------------------|:-------------------|:--------------------------------------------------------------------------------------------------------------|
+| TxHash                 | `string`           | Hash of the project request |
+| SenderDid              | `did.Did`          | Sender account DID          |
+| ProjectDid             | `did.Did`          | Sender's Project DID        |
+| PubKey                 | `string`           | Pubkey of IXO account       |
+| Data                   | `json.RawMessage`  | What the data is passing.   |
+
+
+```go
+type MsgCreateProject struct {
+	TxHash     string          `json:"txHash" yaml:"txHash"`
+	SenderDid  did.Did         `json:"senderDid" yaml:"senderDid"`
+	ProjectDid did.Did         `json:"projectDid" yaml:"projectDid"`
+	PubKey     string          `json:"pubKey" yaml:"pubKey"`
+	Data       json.RawMessage `json:"data" yaml:"data"`
+}
+```
+
+This message is expected to fail if:
+
+- senderDid is wrong  as input
+- PubKey is wrong as input
+
+This message creates and stores the `Project` object at appropriate indexes. 
+
+## MsgUpdateProjectStatus
+
+The owner of a bond can edit some of the bond's parameters using `MsgUpdateProjectStatus`.
```

Comment:
This mentions bonds but we're not dealing with bonds here.

Reply:
removed